crates/next-core/src/next_import_map.rs

```diff
@@ -806,17 +806,15 @@ async fn insert_next_shared_aliases(
 ) -> Result<()> {
     let package_root = next_js_fs().root();
 
-    if next_config.mdx_rs().await?.is_some() {
-        insert_alias_to_alternatives(
```

Comment:
I don't think there's any harm in always enabling this `@vercel/turbopack-next/mdx-import-source` import mapping?